src/GoTrueClient.ts

```diff
@@ -558,6 +562,27 @@ export default class GoTrueClient {
     }
   }
 
+  private async _handleOpenIDConnectSignIn(
+    { IDToken, nonce, clientID, issuer, provider }: OpenIDConnectCredentials
+  ): Promise<{
+    session: Session | null
+    user: User | null
+    error: ApiError | null
+  }> {
+    if (IDToken && nonce && ( (clientID && issuer) || provider) ) {
+      try {
+        const { data, error } = await this.api.signInWithOpenIDConnect({IDToken, nonce, clientID, issuer, provider})
```

Comment:
```suggestion
        const { data, error } = await this.api.signInWithOpenIDConnect({id_token, nonce, client_id, issuer, provider})
```

Reply:
Oh, it might be:


```
this.api.signInWithOpenIDConnect({IDToken: id_token, nonce, clientID: client_id, issuer, provider})
```

for GoTrue on the api side.